src/main/java/kostka/moviecatalog/service/ScheduledService.java

```diff
@@ -21,18 +22,30 @@ public ScheduledService(final RabbitMqSender rabbitMqSender) {
     @Scheduled(initialDelay = INIT_DELAY, fixedDelay = FIXED_DELAY)
     public void refreshLatestMovies() {
         LOGGER.info("[Scheduled task] Refreshing latest movies.");
-        rabbitMqSender.sendToLatestMoviesQueue();
+        try {
+            rabbitMqSender.sendToLatestMoviesQueue();
+        } catch (Exception e) {
+            LOGGER.info(SCHEDULED_TASK_NO_MOVIE_IN_DB);
```

Comment:
SUGGESTION: Pokud logujes exceptions, tak je lepsi pouzit `LOGGER.error(..., e)`. Jednak ti to zaloguje log ktery je typu (levelu) _chyba_ a ne info a jednak to umi zpracovat i tu exception a ulozit ji taky. ;)

Reply:
dobry point!